bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -5,6 +5,15 @@ from operators.task_sensor import ExternalTaskCompletedSensor
 import datetime
 from utils.gcp import bigquery_etl_query, gke_command
 
+{% set ns = namespace(uses_fivetran=False) -%}
+{% for task in tasks -%}
+{% if task.depends_on_fivetran != None and task.depends_on_fivetran|length > 1 and ns.uses_fivetran == False -%}
+{% set ns.uses_fivetran = True -%}
```

Comment:
Just curious, what's `ns` used for here? I haven't used jinja namespaces

Reply:
I wanted to make sure that the Fivetran related imports are added exactly once if there actually Fivetran dependencies. Simply using `{% set uses_fivetran = false -%}` does not work due to Jinjas [scoping behaviour](https://jinja.palletsprojects.com/en/2.10.x/templates/#assignments). Even when setting `{% set uses_fivetran = true -%}` inside the loop, it will still show up as `false` outside of the loop. Using namespaces propagates changes across scopes.